lib/matplotlib/tests/test_mathtext.py

```diff
@@ -385,3 +385,35 @@ def test_math_fontfamily():
              size=24, math_fontfamily='dejavusans')
     fig.text(0.2, 0.3, r"$This\ text\ should\ have\ another$",
              size=24, math_fontfamily='stix')
+
+def test_default_math_fontfamily():
+    mpl.rcParams['mathtext.fontset'] = 'cm'
+    test_str = r'Foo: $\alpha_{i+1}^j = {\rm sin}(2\pi f_j t_i) e^{-5 t_i/\tau}$'
```

Comment:
Does this test string really need to be so long? Isn't it just the properties of the text object that matter?

Reply:
I just used an example test string at the start of this file. The requirement is that it should have some math text (quoted in $$) and normal text.